test/core/event_engine/test_suite/fuzzing_event_engine_test.cc

```diff
@@ -29,12 +32,15 @@ class ThreadedFuzzingEventEngine : public FuzzingEventEngine {
   ThreadedFuzzingEventEngine()
       : FuzzingEventEngine([]() {
           Options options;
-          options.final_tick_length = absl::Milliseconds(10);
+          options.final_tick_length = std::chrono::milliseconds(10);
           return options;
         }()),
         main_([this]() {
           while (!done_.load()) {
-            absl::SleepFor(absl::Milliseconds(10));
+            auto tick_start = absl::Now();
+            while (absl::Now() - tick_start < absl::Milliseconds(10)) {
+              absl::SleepFor(absl::Milliseconds(1));
```

Comment:
Is there  a problem with `absl::SleepFor(absl::Milliseconds(10));`??

Reply:
it was triggering a little bit early in some cases